packages/module/src/ContentHeader/ContentHeader.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { Divider, Flex, FlexItem, Icon, Label, Split, SplitItem, Text } from '@patternfly/react-core';
+import { ExternalLinkAltIcon } from '@patternfly/react-icons';
+import ActionMenu, { ActionMenuProps, ActionMenuVariant } from '../ActionMenu/ActionMenu';
+
+export interface ContentHeaderProps {
+/** title for content header */ 
+  title: string;
+/** subtitle for content header */
+  subtitle: string;
+/** Optional link below subtitle */
+  link?: string;
+/** Optional icon for content header (appears to the left of the content header's title with a divider) */
+  icon?: React.ReactNode
+/** Optional badge for content header (appears to the right of the content header's title) */
+  label?: React.ReactNode;
+/** Breadcrumbs component */
+  breadcrumbs?: React.ReactNode;
+/** Menu that appears to the far right of the title */
+  actionMenu?: ActionMenuProps;
+  /** Custom OUIA ID */
+  ouiaId?: string | number,
+}
+
+const ContentHeader: React.FunctionComponent<React.PropsWithChildren<ContentHeaderProps>> = ({
```

Comment:
we want the named export as well
```suggestion
export const ContentHeader: React.FunctionComponent<React.PropsWithChildren<ContentHeaderProps>> = ({
```